src/RpcClient/Utility.cs

```diff
@@ -16,6 +16,8 @@ namespace Neo.Network.RPC
 {
     public static class Utility
     {
+        internal static ProtocolSettings ProtocolSettings = ProtocolSettings.Load("protocol.json");
```

Comment:
I think we should add `ProtocolSettings` to `RpcClient`. Otherwise, the SDK can only support a single network.

Reply:
Done.